nss-cohort-36/keahua-arboretum-pixel-pals
# Lr animal release bug fixes

# Description

User should be able to create animals and add them to biomes

Fixes #39 (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

1. `git fetch --all`
2. `git checkout lr-animal-release-bug-fixes`
3. `python index.py`
4. Create one of every biome and then add one of each animal to a biome. There should be no errors. Note: logic to not show a biome that is full has not been added yet.



# Checklist:
